daemon/indexing/apollo/index.js

```diff
@@ -19,11 +19,11 @@ const typeDefs = gql`
   ######################
 
   # When querying a set of items, the output is a page.
- # interface OutputPage {
- #   pageNumber: Int!
- #   itemsPerPage: Int!
- #   totalNumberOfPages: Int!
- # }
+  interface OutputPage {
+    pageNumber: Int!
+    itemsPerPage: Int!
+    totalNumberOfItems: Int!
```

Comment:
If we decide to go with totalNumberOfItems then I feel we should be consistent and use items vs pages for the whole interface. Something like this:

```
interface InputPage {
  offset: Int!
  numItems: Int!
}

interface OutputPage {
  offset: Int!
  numItems: Int!
  totalNumItems: Int!
}
```

Reply:
Yeah you are right, I will go with what you propose, since the pages approach isn't good enough. It doesn't communicate the total amount of items and we display that information in the Dapp.